skimage/io/tests/test_fits.py

```diff
@@ -23,44 +23,10 @@ def teardown():
     io.reset_plugins()
 
 
-def test_imread_MEF():
-    io.use_plugin('fits')
-    testfile = os.path.join(data_dir, 'multi.fits')
-    img = io.imread(testfile)
-    assert np.all(img == fits.getdata(testfile, 1))
-
-
-def test_imread_simple():
-    io.use_plugin('fits')
-    testfile = os.path.join(data_dir, 'simple.fits')
-    img = io.imread(testfile)
-    assert np.all(img == fits.getdata(testfile, 0))
-
-
-def test_imread_collection_single_MEF():
-    io.use_plugin('fits')
-    testfile = os.path.join(data_dir, 'multi.fits')
-    ic1 = io.imread_collection(testfile)
-    ic2 = io.ImageCollection(
-        [(testfile, 1), (testfile, 2), (testfile, 3)],
-        load_func=fplug.FITSFactory)
-    assert _same_ImageCollection(ic1, ic2)
-
-
-def test_imread_collection_MEF_and_simple():
-    io.use_plugin('fits')
-    testfile1 = os.path.join(data_dir, 'multi.fits')
-    testfile2 = os.path.join(data_dir, 'simple.fits')
-    ic1 = io.imread_collection([testfile1, testfile2])
-    ic2 = io.ImageCollection([(testfile1, 1), (testfile1, 2),
-                             (testfile1, 3), (testfile2, 0)],
-                             load_func=fplug.FITSFactory)
-    assert _same_ImageCollection(ic1, ic2)
-
-
 def _same_ImageCollection(collection1, collection2):
-    """Ancillary function to compare two ImageCollection objects, checking
-       that their constituent arrays are equal.
+    """
+    Ancillary function to compare two ImageCollection objects, checking that
+    their constituent arrays are equal.
```

Comment:
If the FITS plugin is now broken, this should *definitely* be noted in the release notes.

Reply:
Indeed. I'll check